test/integration/builder/additional.js

```diff
@@ -175,6 +175,9 @@ module.exports = function(knex) {
           // Insert new data after truncate and make sure ids restart at 1.
           // This doesn't currently work on oracle, where the created sequence
           // needs to be manually reset.
+          // On redshift, one would need to create an entirely new table and do
```

Comment:
This block is in a bit strange position separating comment about oracle from if()... maybe skipping this for oracle could be also implemented the same way that is is implemented for redshift.